### DISCUSSION TITLE
Finalize member application

### DISCUSSION THREAD

**Discussion by enrico-ghidoni** [CntoDev/level-up#112] [General]
Members (interviewers only when branch membership will be defined, outside MVP) can either accept or refuse a submitted application form.

In case of acceptance: a new member for the applicant is created, recruit rank is assigned (when rank system will be defined, outside MVP), community services eligible for automation are updated (i.e. applicant is automatically added the tag of recruit on Discord).

**Comment by milivojm** [comment]
- [x] accept application
- [x] reject application
- [x] create member on application acceptance
- [x] assign recruit rank
- [ ] give permissions on teamspeak
- [x] give permissions on discord
- [ ] give permissions on website

This issue is an [Epic](https://www.agilealliance.org/glossary/epic/#q=~(infinite~false~filters~(postType~(~'page~'post~'aa_book~'aa_event_session~'aa_experience_report~'aa_glossary~'aa_research_paper~'aa_video)~tags~(~'epic))~searchTerm~'~sort~false~sortDirection~'asc~page~1)). I'd advise to split it into multiple smaller issues and track them here.

**Comment by milivojm** [comment]
Accept application, reject application, create member on application acceptance done.

**Comment by enrico-ghidoni** [comment]
So do you wanna keep just this is issue as reference for the multiple tasks?

**Comment by milivojm** [comment]
> So do you wanna keep just this is issue as reference for the multiple tasks?

Maybe yes.

**Comment by milivojm** [comment]
Assign recruit rank done. Now the fun part -> Skynet. Shall we open separate issues for Teamspeak and Website?

**Comment by enrico-ghidoni** [comment]
Yeah they're gonna be open for a long time (especially the website one)

**Comment by milivojm** [comment]
Moving this to discussions.
